examples/advanced/job_config/hello-pt/add_shareable_parameter.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from nvflare.apis.filter import Filter
+from nvflare.apis.fl_context import FLContext
+from nvflare.apis.shareable import Shareable
+
+
+class AddShareable(Filter):
+    def process(self, shareable: Shareable, fl_ctx: FLContext) -> Shareable:
+        print(f"{fl_ctx.get_identity_name()} ---- AddShareable Filter ----")
```

Comment:
This is just printing, should we rename to PrintShareable?

Reply:
There's already another PrintShareable in the example. Keep this name.